.travis.yml

```diff
@@ -10,7 +10,10 @@ install:
   - pip install -r requirements.txt
   - npm i -g npm
   - npm i -g serverless
+  - npm i serverless-python-requirements
+  - pip install pipenv
 # command to run tests
 script:
   # - sls deploy
-  - echo "Doing nothin now"
+  - pytest
+  - serverless deploy
```

Comment:
Add this command before deploy
`serverless config credentials --provider aws --key $aws_access_key_id --secret $aws_secret_access_key --profile iam-aws`

Reply:
Sure

Reply:
@gurungrahul2 on it?

Reply:
@aashutoshrathi done.